utils.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+
+function writeTextToFile(contents, filePath) {
+  try {
+    fs.writeFileSync(filePath, contents);
```

Comment:
added methods as synchronous style, we can change to async if required. Thought it would be easy instead of handling callbacks and await given less time we have :-p

Reply:
Can we also write a method to save image.

Reply:
yeah sure, will add them too